### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by xilopaint**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,48 +2,49 @@
...
+      "subtitle": "Raycast Charts",
+      "description": "Charts ranked by installation count of extensions authors",
       "mode": "view"
     },
     {
       "name": "store",
-      "title": "Search Store",
-      "description": "Show the store",
+      "title": "Search Raycast Store",
+      "subtitle": "Raycast Charts",
```
Comment (tonka3000) [inline package.json]: Remove the subtitle here, it get inherited from the package title.

File: package.json [hunk 2]
```diff
@@ -2,48 +2,49 @@
...
+      "description": "Show extensions that are new or have been updated",
       "mode": "view"
     },
     {
       "name": "author_charts",
       "title": "Author Charts",
-      "subtitle": "Raycast",
-      "description": "Charts ranked by install count of an authors extensions in total",
+      "subtitle": "Raycast Charts",
```
Comment (tonka3000) [inline package.json]: Remove the subtitle here, it get inherited from the package title.

File: package.json [hunk 3]
```diff
@@ -2,48 +2,49 @@
...
+      "description": "Charts ranked by extension installation count",
       "mode": "view"
     },
     {
       "name": "extensions_news",
       "title": "Extension News",
-      "subtitle": "Raycast",
-      "description": "Show Extension which are new or get updated",
+      "subtitle": "Raycast Charts",
```
Comment (tonka3000) [inline package.json]: Remove the subtitle here, it get inherited from the package title.

File: package.json [hunk 4]
```diff
@@ -2,48 +2,49 @@
...
   "icon": "analytics.png",
   "author": "tonka3000",
   "license": "MIT",
   "commands": [
     {
       "name": "extension_charts",
       "title": "Extension Charts",
-      "subtitle": "Raycast",
-      "description": "Charts ranked by extension install count",
+      "subtitle": "Raycast Charts",
```
Comment (tonka3000) [inline package.json]: Remove the subtitle here, it get inherited from the package title.

**Comment by xilopaint** [comment]
Done.

**Pull request merged**